test/normalize.py

```diff
@@ -66,7 +67,7 @@ def handle_starttag(self, tag, attrs):
                     self.output += ("=" + '"' +
                             urllib.quote(urllib.unquote(v), safe='/') + '"')
                 elif v != None:
-                    self.output += ("=" + '"' + cgi.escape(v,quote=True) + '"')
```

Comment:
Nit: we're still importing `cgi` even though we're no longer using it

Reply:
Looks like the implementation in #216 is more defensive, it we want to support multiple Python versions.